src/components/Atoms/Navigation.tsx

```diff
@@ -0,0 +1,64 @@
+import { MenuItem } from '@mui/material';
+import Link from '@mui/material/Link';
+import { NavLink } from 'react-router-dom';
+
+import { ROUTES } from '../../routes/Routes';
+
+type NAV_ITEM = {
```

Comment:
Type should start just with capital letter, what you did here is convention for constants